include/sqlpp11/mysql/bind_result.h

```diff
@@ -44,11 +44,11 @@ namespace sqlpp
     class bind_result_t
     {
       std::shared_ptr<detail::prepared_statement_handle_t> _handle;
-      void* _result_row_address = nullptr;
+      void* _result_row_address {nullptr};
```

Comment:
Here in the other places, please omit the space between variable and brace initializer (clang-format).

Reply:
OK, I removed the spaces, run it through clang-format just to confirm that it no longer complains and then re-ran the tests again. It should be ready for a review again.

Reply:
Thanks!